src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -166,31 +166,19 @@ void TLayout::layoutItem(EngravingItem* item, LayoutContext& ctx)
 
     switch (item->type()) {
     case ElementType::ACCIDENTAL:
-        if (!ldata->isValid()) {
-            layout(item_cast<const Accidental*>(item), static_cast<Accidental::LayoutData*>(ldata), ctx.conf());
-        }
+        layout(item_cast<const Accidental*>(item), static_cast<Accidental::LayoutData*>(ldata), ctx.conf());
```

Comment:
About this commit: as long as we have `PassResetLayoutData`, the caching won't do much, right? And a prerequisite for removing that pass, is that we properly invalidate layout data wherever that needs to happen.

Reply:
That’s right, caching between layout calls is not currently provided, that is, I’m not considering it yet (so as not to deal with all the complexity at once, but do it step by step)
But now there is caching so that the same element is not layout several times during the layout process (for example, now this can definitely happen for `independent` ones)